src/patternfly/components/Masthead/masthead.scss

```diff
@@ -0,0 +1,234 @@
+  }
+
+  @include pf-t-dark; // force the container to follow the dark theme
+
+  position: relative;
+  display: flex;
+  flex-wrap: var(--pf-c-masthead--FlexWrap);
+  align-items: center;
+  min-width: 0;
+  padding-right: var(--pf-c-masthead--PaddingRight);
+  padding-left: var(--pf-c-masthead--PaddingLeft);
+  background-color: var(--pf-c-masthead--BackgroundColor);
+
+  &.pf-m-light-100 {
+    @include pf-t-light;
+
+    --pf-c-masthead--BackgroundColor: var(--pf-c-masthead--m-light-100--BackgroundColor);
+    --pf-c-masthead__main--BorderBottomColor: var(--pf-c-masthead--m-light-100__main--BorderBottomColor);
+  }
+
+  &.pf-m-light-200 {
+    @include pf-t-light;
+
+    --pf-c-masthead--BackgroundColor: var(--pf-c-masthead--m-light-200--BackgroundColor);
+    --pf-c-masthead__main--BorderBottomColor: var(--pf-c-masthead--m-light-200__main--BorderBottomColor);
+  }
+
+  &.pf-m-page-insets {
+    --pf-c-masthead--inset: var(--pf-c-masthead--m-page-insets--inset);
+  }
+}
+
+// main
+.pf-c-masthead__main {
+  position: relative;
+  display: flex;
+  flex-basis: var(--pf-c-masthead__main--FlexBasis);
+  align-items: center;
+  align-self: stretch;
+  order: var(--pf-c-masthead__main--Order);
+  padding-top: var(--pf-c-masthead__main--PaddingTop);
+  padding-bottom: var(--pf-c-masthead__main--PaddingBottom);
+  margin-right: var(--pf-c-masthead__main--MarginRight);
+
+  &::before {
+    position: absolute;
+    right: var(--pf-c-masthead__main--before--Right);
+    bottom: 0;
+    left: var(--pf-c-masthead__main--before--Left);
+    content: "";
+    border-bottom: var(--pf-c-masthead__main--before--BorderBottom);
+  }
+
+  &:last-child {
+    --pf-c-masthead__main--MarginRight: 0;
+  }
+}
+
+// content
+.pf-c-masthead__content {
+  display: flex;
+  flex-grow: 1;
+  flex-shrink: 1;
+  align-items: center;
+  align-self: stretch;
+  order: var(--pf-c-masthead__content--Order);
+  padding-top: var(--pf-c-masthead__content--PaddingTop);
+  padding-bottom: var(--pf-c-masthead__content--PaddingBottom);
+  margin-left: var(--pf-c-masthead__content--MarginLeft);
+
+  &:only-child {
+    --pf-c-masthead__content--MarginLeft: 0;
+  }
+
+  .pf-c-nav.pf-m-horizontal {
+    margin-right: var(--pf-c-masthead__content--c-nav--m-horizontal--MarginRight);
+    margin-left: var(--pf-c-masthead__content--c-nav--m-horizontal--MarginLeft);
+  }
+
+  .pf-c-masthead__toggle ~ & {
+    --pf-c-masthead__content--c-nav--m-horizontal--MarginLeft: calc(var(--pf-c-masthead__content--MarginLeft) * -1);
+  }
+}
+
+.pf-c-masthead__main,
+.pf-c-masthead__brand,
+.pf-c-masthead__content {
+  min-width: 0; // allow flex containers to shrink to fit possible overflow
+}
+
+// brand
+.pf-c-masthead__brand {
+  display: inline-flex;
+  align-self: center;
+}
+
+// toggle
+.pf-c-masthead__toggle {
+  align-self: center;
+  order: 0;
+  margin-right: var(--pf-c-masthead__toggle--MarginRight);
+
+  .pf-c-button {
+    margin-left: calc(var(--pf-c-button--PaddingLeft) * -1);
```

Comment:
This works but usually we'll just offset the component element (`.pf-c-masthead__toggle`). The reasoning was, while this updates automatically if we change `--pf-c-button--PaddingLeft`, it breaks if we change `--pf-c-button--PaddingLeft` (we changed some var names and broke some rules like this). And it's more inline with keeping the styling for a component in its component. And for having a 1:1 with a style property to component var (ie, `margin-left: var(--pf-c-masthead__toggle--MarginLeft);` that's themable/overridable.